scripts/west_commands/zcmake.py

```diff
@@ -260,3 +263,26 @@ def __delitem__(self, name):
 
     def __iter__(self):
         return iter(self._entries.values())
+
+def _ensure_min_version(cmake):
+    try:
+        version_out = subprocess.check_output([cmake, '--version'],
+                                              stderr=subprocess.DEVNULL)
+    except subprocess.CalledProcessError as cpe:
+        log.die('cannot get cmake version:', str(cpe))
+    decoded = version_out.decode('utf-8')
+    lines = decoded.splitlines()
+    if not lines:
+        log.die('can\'t get cmake version:',
+                'unexpected "cmake --version" output:\n{}'.
+                format(decoded))
+    version = lines[0].split()[2]
+    if packaging.version.parse(version) < _MIN_CMAKE_VERSION:
+        log.die('cmake version', version, 'is less than minimum',
+                _MIN_CMAKE_VERSION_STR)
+    else:
+        log.dbg('cmake version', version, 'is OK; minimum version is',
+                _MIN_CMAKE_VERSION_STR)
+
+_MIN_CMAKE_VERSION_STR = '3.13.1'
```

Comment:
Could we instead get this from the boilerplate file? At this point we must’ve access to it?

Reply:
Yeah, I have thought about it, but that version number is hard-coded across every single sample in this repository, so I don't think one more instance would really provide any benefit.

Reply:
Very well, let's leave it as-is then.